samples/concepts.js

```diff
@@ -16,12 +16,13 @@
 'use strict';
 
 const sinon = require('sinon');
+const assert = require('assert');
 
 // By default, the client will authenticate using the service account file
 // specified by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use
 // the project specified by the GCLOUD_PROJECT environment variable. See
 // https://googlecloudplatform.github.io/gcloud-node/#/docs/google-cloud/latest/guides/authentication
-const {Datastore} = require('@google-cloud/datastore');
```

Comment:
This was correct the way it was :)

Reply:
Side note - make sure you're running `npm link ../` from inside of the `samples` directory.  That ensures you use the sources instead of the published module to try this out ;)

Reply:
Thanks @JustinBeckwith ,
Change done.